### PR TITLE
Go to file by entering directly the path (Library / File / Member)

### PR DESCRIPTION
**Pull request by sebCIL**
### Changes

Go to file by entering directly the path (Library / File / Member)
185429220-15e2159b-5045-4509-9a62-3cd00035ccd4 1
185428960-6b20c9c9-ec5f-4114-b5b8-3acd247a6d7a 1
185429000-ec44cc9a-461a-4c6f-979b-f23639f79ffe 1

### Checklist

* [X] have tested my change
* [ ] updated relevant documentation
* [X] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [ ] have added myself to the contributors' list in [CONTRIBUTING.md](https://github.com/halcyon-tech/vscode-ibmi/blob/master/CONTRIBUTING.md)
* [ ] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

**Review by worksofliam — CHANGES_REQUESTED** [review]
This is a great PR. You have basically reimplemented `code-for-ibmi.openFileByPath` by allowing a list and user-input, so here is what I think needs to happen:

* We remove `code-for-ibmi.openFileByPath` completely
* We update the keybinding to `code-for-ibmi.openFileByPath` to point to `code-for-ibmi.goToFile` so the keybinding continue to work.
* The new `quickInput` needs a `prompt` value if it is possible. Something like: 'Select file or enter path' (see the prompt in `openFileByPath`)

Let me know if you can do this. Thank you!

**Comment by sebCIL** [comment]
Very simple but it works.

Not sure to understand ...

My modifications was in the `code-for-ibmi.goToFile` command not in `code-for-ibmi.openFileByPath`.
I have put a placeholder with  ```quickPick.placeholder = `Go to file..`;``` I can change it to what you want.

**Comment by worksofliam** [comment]
@sebCIL Since `code-for-ibmi.goToFile` and `code-for-ibmi.openFileByPath` are the same now, I am suggesting we remove `code-for-ibmi.openFileByPath`. 

What do you think of this?

**Comment by chrjorgensen** [comment]
Is it (still?) possible to open a file in the IFS using this - or is it limited to QSYS filesystem?

**Comment by worksofliam** [comment]
@chrjorgensen It supports both IFS and QSYS.

**Comment by chrjorgensen** [comment]
@worksofliam @sebCIL Then we just need to make sure that this is reflected by the prompt or placeholder - that both member and streamfile can be specified.

**Comment by sebCIL** [comment]
Ok, I made some changes.

**Review by worksofliam — CHANGES_REQUESTED** [review]
@sebCIL please check like 403 in `Instance.js`. Looks like you've added an additional closing bracket which doesn't need to be there. Then please run eslint for cleanup afterwards.

This is looking great otherwise!

File: src/Instance.js [hunk 1]
```diff
@@ -393,24 +374,31 @@ module.exports = class Instance {
...
-            } else {
-              vscode.window.showErrorMessage(`No files to select from.`);
-            }
+              }
+              quickPick.hide()
+            })
+            quickPick.onDidHide(() => quickPick.dispose());
+            quickPick.show();
           })
         )
```
Comment (worksofliam) [inline src/Instance.js:403]: Please remove this closing bracket.

Comment (sebCIL) [reply src/Instance.js]: Oops ....
I remove it and make a eslint review.

**Review by worksofliam — APPROVED** [review]
Tested:

* Opening a member from list
* Opening a streamfile from list
* Opening member from string
* Opening streamfile from string

Great change!

**Pull request merged**